utils/firestore.js

```diff
@@ -2201,72 +2201,73 @@ const setPackageReceiptUSPS = async (data) => {
     try {
         const { bagConceptIDs } = require('./shared');
-        let token = data.scannedBarcode
+        let trackingNumber = data.scannedBarcode
         let collectionIdHolder = {}
-        if ((token).length === 34) token = data.scannedBarcode.slice((data.scannedBarcode).length - 22)
-        const snapshot = await db.collection("boxes").where('959708259', '==', token).get(); // find related box using barcode
-        if (snapshot.empty) {
-            return false
-        }
+        if ((trackingNumber).length === 34) trackingNumber = data.scannedBarcode.slice((data.scannedBarcode).length - 22)
+        const snapshot = await db.collection("boxes").where('959708259', '==', trackingNumber).get(); // find related box using tracking number
+        
+        if (snapshot.empty) return false;
+        
         const docId = snapshot.docs[0].id;
-        data['959708259'] = token
+        data['959708259'] = trackingNumber
         delete data.scannedBarcode
-        await db.collection("boxes").doc(docId).update(data)
+
+        await db.collection("boxes").doc(docId).update(data); // using the docids update the box with the received date
         
-        // Updated BagConceptIDs for new 40 bag capacity. Retained these because they were previously included.
-        const additionalBagItems = ["255283733", "842312685", "234868461", "522094118"];
-
-        const allBagsToCheck = [...bagConceptIDs, ...additionalBagItems];
-        if (Object.keys(snapshot.docs.length) !== 0) {
-             snapshot.docs.map(doc => { 
-                const collectionIdKeys = doc.data(); // grab all the collection ids
-                allBagsToCheck.forEach(async (bag) => {
-                    if (bag in collectionIdKeys){
-                        if (collectionIdKeys[bag]['787237543'] !== undefined || collectionIdKeys[bag]['223999569'] !== undefined || collectionIdKeys[bag]['522094118'] !== undefined) {
-                            if (collectionIdKeys[bag]['787237543'] !== `` && collectionIdKeys[bag]['223999569'] === `` && collectionIdKeys[bag]['522094118'] === ``) {
-                                collectionIdHolder[bag] = collectionIdKeys[bag]['787237543'].split(' ')[0]
-                            }
-                            if (collectionIdKeys[bag]['223999569'] !== `` && collectionIdKeys[bag]['787237543'] === `` && collectionIdKeys[bag]['522094118'] === ``) {
-                                collectionIdHolder[bag] = collectionIdKeys[bag]['223999569'].split(' ')[0]
-                            }
-                            if (collectionIdKeys[bag]['522094118'] !== `` && collectionIdKeys[bag]['223999569'] === `` && collectionIdKeys[bag]['787237543'] === `` ) {
-                                collectionIdHolder[bag] = collectionIdKeys[bag]['522094118'].split(' ')[0]
-                            }
+        for (const doc of snapshot.docs) {
+            const collectionIdKeys = doc.data(); // grab all the collection ids
```

Comment:
'docData' or 'boxData' can be a better name.

Reply:
Updated to 'boxData'. Thanks.